rpc/gnmi/mixed-schema.md

```diff
@@ -148,10 +166,14 @@ replace: <
 >
 ```
 
-This transaction replaces the contents of both the "openconfig" and "cli"
-origins.  The first `replace` message replaces the OpenConfig origin at the root
-(as specified by the zero-length array of `PathElem` messages) to the specified
-JSON. The second `replace` replaces the CLI configuration of the device is
-replaced with the string specified in the `ascii_val` field. Per the gNMI
-specification, both `replace` operations MUST successfully be applied, otherwise
-the configuration change should be rolled back.
+This transaction replaces the contents of the "openconfig" origins and then
+updates the configuration with CLI content.  The first `replace` message
```

Comment:
This still references `replace` on the CLI -- which I think is correct, but you disallowed it above :-)

Reply:
I think I resolved this conflict now.  :)